plugin/src/main/java/hudson/plugins/findbugs/FindBugsPublisher.java

```diff
@@ -154,6 +164,21 @@ public boolean isRankActivated() {
     }
 
     /**
+     * Added to properly uncoercing.
```

Comment:
It was added because (I thought) it's not obvious that this method is only needed to have the workflow Snippetizer working fine (the method is not used anywhere in the project actually)